src/Forms/RequiredFields.php

```diff
@@ -122,6 +126,10 @@ public function php($data)
                 }
             } else {
                 $stringValue = (string) $value;
+                if (!static::config()->get('allow_whitespace_only')) {
+                    $stringValue = preg_replace('/^\s+/u', '', $stringValue);
+                    $stringValue = preg_replace('/\s+$/u', '', $stringValue);
+                }
```

Comment:
Should also be configurable per validator instance via a `setter` as per the acceptance criterion:

> This should be configurable globally via YAML, _and configurable for individual RequiredFieldsValidator instances via a new setter method._

Reply:
Done